### PR TITLE
Renaming TEMP to dashboards

### LINKED ISSUE #3534: Renaming gobierto_TEMP to gobierto_dashboards
Apply the new name. Paths:
- [ ] `app/stylesheets`
- [ ] `app/javascript`
- [ ] `app/controllers/gobierto_admin`
- [ ] `app/views/gobierto_admin`
- [ ] `config/routes.rb`
- [ ] `app/sandbox`

### PR DESCRIPTION
**Pull request by Crashillo**
## :v: What does this PR do?
Rename TEMP elements to dashboards

- admin: https://madrid.gobify.net/sandbox/admin_dashboards
- dashboard: https://madrid.gobify.net/sandbox/dashboards

### CODE REVIEW

File: app/assets/stylesheets/module-dashboards-maker.scss [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+    }
+  }
+
+  &--button {
+    @include flex-align(space-between, baseline);
+    @include border-radius(4px);
+
+    font-size: $f8;
+    font-family: var(--font-family);
+    line-height: 15px;
```
Comment (jorgeatgu) [inline app/assets/stylesheets/module-dashboards-maker.scss]: If you can, it's best practice to use unitless numbers for line-height values

[Old but gold](http://meyerweb.com/eric/thoughts/2006/02/08/unitless-line-heights/)

File: app/assets/stylesheets/module-dashboards-viewer.scss [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+      &--title {
+        font-weight: bold;
+        font-size: $f7;
+        color: var(--color-base);
+        opacity: 0.75;
+      }
+
+      &--value {
+        font-size: $f1;
+        line-height: 53px;
```
Comment (jorgeatgu) [inline app/assets/stylesheets/module-dashboards-viewer.scss]: same above

**Comment by ferblape** [comment]
The assets aren't compiled in CircleCI https://app.circleci.com/pipelines/github/PopulateTools/gobierto/2721/workflows/727305a1-b951-4a7b-9b5f-f5044aecca14/jobs/23751 worth to investigate the error

**Comment by ferblape** [comment]
Assets compiled again, it was a glitch from CircleCI

**Review by ferblape — APPROVED** [review]
OK to me

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script>
+import { date, percent } from "lib/shared";
+
+const formatValues = (arr, prop, transform = x => x) =>
```
Comment (jorgeatgu) [inline app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue]: Maybe you can extract this function in a new file `/lib/utils.js` what do you think?

**Review by jorgeatgu — APPROVED** [review]

File: app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue (hunk 3 above)
Comment (Crashillo) [reply app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue]: Mmmm, very specific function. Only applies to this component, in order to generate a particular object.
Besides, it has an internal dependency (`date` function), so better keep it right there.

File: app/assets/stylesheets/module-dashboards-maker.scss (hunk 1 above)
Comment (Crashillo) [reply app/assets/stylesheets/module-dashboards-maker.scss]: Done both

**Pull request merged**